src/component/visualMap/ContinuousView.ts

```diff
@@ -191,7 +192,8 @@ class ContinuousView extends VisualMapView {
                 align: orient === 'horizontal' ? align as TextAlign : 'center',
                 text: text,
                 font: textStyleModel.getFont(),
-                fill: textStyleModel.getTextColor()
+                fill: textStyleModel.getTextColor(),
+                ...textShadow
```

Comment:
If you hope more styles get applied, I think it may be better to use `createTextStyle` in `src/label/labelStyle.ts`.

Reply:
you are right, `createTextStyle` is more powerful.